Source/WebCore/dom/ElementRareData.h

```diff
@@ -42,6 +42,7 @@
 #include "StylePropertyMap.h"
 #include "StylePropertyMapReadOnly.h"
 #include "VisibilityAdjustment.h"
+#include "WritingSuggestionData.h"
```

Comment:
Why do we need to add this include? I don’t see any new code in this file. This must be left over from an earlier version of this patch.